pygmt/src/velo.py

```diff
@@ -0,0 +1,248 @@
+"""
+velo - Plot velocity vectors, crosses, anisotropy bars and wedges.
+"""
+import numpy as np
+import pandas as pd
+from pygmt.clib import Session
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import build_arg_string, fmt_docstring, kwargs_to_strings, use_alias
+
+
+@fmt_docstring
+@use_alias(
+    A="vector",
+    B="frame",
+    C="cmap",
+    D="rescale",
+    E="uncertainty_color",
+    G="facecolor",
+    H="scale",
+    I="intensity",
+    J="projection",
+    L="line",
+    N="no_clip",
+    R="region",
+    S="scaling",
+    U="timestamp",
+    V="verbose",
+    W="pen",
+    X="xshift",
+    Y="yshift",
+    Z="zvalue",
+    c="panel",
+    p="perspective",
+    t="transparency",
+)
+@kwargs_to_strings(R="sequence", c="sequence_comma", i="sequence_comma", p="sequence")
+def velo(self, data=None, **kwargs):
+    r"""
+    Plot velocity vectors, crosses, anisotropy bars and wedges.
```

Comment:
```suggestion
    Plot velocity vectors, crosses, anisotropy bars, and wedges.
```

I'm not trying to push my opinions about serial commas on the project; just making this suggestion because all the other docstrings in https://pygmt-echqpso25-gmt.vercel.app/api/index.html include them.

Reply:
Personally, I do like serial/Oxford commas. Upstream GMT's `velo` purpose doesn't have a serial comma (https://github.com/GenericMappingTools/gmt/blob/f24ed59277bcf03b321f4f8de6a8bf6077c384f0/doc/rst/source/supplements/module_supplements_purpose.rst_#L7), but I do see other places having it, so I'll add it in.